comp/otelcol/otlp/components/exporter/serializerexporter/factory.go

```diff
@@ -79,13 +168,22 @@ func (f *factory) createMetricExporter(ctx context.Context, params exp.Settings,
 		exporterhelper.WithTimeout(cfg.TimeoutConfig),
 		// the metrics remapping code mutates data
 		exporterhelper.WithCapabilities(consumer.Capabilities{MutatesData: true}),
```

Comment:
Note: if you use this pattern, be careful to not apply `WithQueue` `WithTimeout` etc. again in the OSS datadog exporter metrics exporter factory